samples/visual_pane/visual_pane.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://aframe.io/releases/1.1.0/aframe.min.js"></script>
+    <script src="https://unpkg.com/aframe-drag-controls@1.0.0/index.js"></script>
+    <script src="components/visual_pane.js"></script>
+  </head>
+  <body>
+    <a-scene
+      background="color: #AAD5D7"
+      cursor="rayOrigin: mouse"
+      raycaster="objects: .link"
+    >
+      <a-assets>
+        <img id="visual1" src="./pic2.jpg" crossorigin="anonymous" />
```

Comment:
`pic2.jpg` isn't included in this PR, but `pic1.png` is, does this need to be updated? 😄

Reply:
oh yes, ill change back to `pic1.png`